src/Core/CMakeLists.txt

```diff
@@ -23,21 +23,47 @@ target_include_directories(ObEngineCore
         $<BUILD_INTERFACE:${ObEngine_BINARY_DIR}/include/Core>
 )
 
+if(OBE_USE_VCPKG)
+    if (USE_FILESYSTEM_FALLBACK)
+        target_include_directories(ObEngineCore ${TINYDIR_INCLUDE_DIRS})
+    endif()
+
+    if(WIN32)
+        target_link_libraries(ObEngineCore FLAC OpenAL OpenGL Vorbis)
+    else()
+        target_link_libraries(ObEngineCore X11 FLAC UDev OpenAL)
+    endif()
+
+
+    target_link_libraries(ObEngineCore ${LUA_LIBRARIES})
+    target_include_directories(ObEngineCore ${LUA_INCLUDE_DIR})
+
+    target_link_libraries(ObEngineCore sol2::sol2)
+    target_link_libraries(ObEngineCore fmt::fmt fmt::fmt-header-only)
```

Comment:
Is there a non-header-only version available ?

Reply:
I think vcpkg supports only the header-only version.

Reply:
At least, on my pc

Reply:
It seems you are linking the two versions of the library, the standard one and the header only version.
Check out this link : https://fmt.dev/dev/usage.html#usage-with-cmake
Does removing the `fmt::fmt-header-only` and keeping `fmt::fmt` works ?

Reply:
Oh, I thought you were talking about sol2. I guess fmt will still work, there are places on my project where I use `fmt/core.h`, or `fmt/format.h` alone, so there are multiple headers.

Reply:
Same for `spdlog` :) I'd rather use the non-header-only version for faster build times